### PR TITLE
CB-3078 te secret events

### PR DESCRIPTION
**Pull request by yagudin10**

### CODE REVIEW

File: server/bundles/io.cloudbeaver.server/src/io/cloudbeaver/server/events/WSUserSecretEventHandlerImpl.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import org.jkiss.dbeaver.model.websocket.event.datasource.WSDataSourceProperty;
+
+import java.util.List;
+
+/**
+ * Notify all active user session that rm resource has been updated
+ */
+public class WSUserSecretEventHandlerImpl extends WSProjectUpdatedEventHandler {
+
+    private static final Gson gson = new GsonBuilder().create();
```
Comment (alexander-skoblikov) [inline server/bundles/io.cloudbeaver.server/src/io/cloudbeaver/server/events/WSUserSecretEventHandlerImpl.java]: unused?

File: server/bundles/io.cloudbeaver.server/src/io/cloudbeaver/server/events/WSUserSecretEventHandlerImpl.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        return WSEventTopic.USER_SECRET.getTopicId();
+    }
+
+    @Override
+    protected void updateSessionData(BaseWebSession activeUserSession, WSEvent event) {
+        if (!(event instanceof WSUserSecretEvent)) {
+            return;
+        }
+        var resourceUpdateEvent = (WSUserSecretEvent) event;
+        if (activeUserSession instanceof WebSession) {
```
Comment (alexander-skoblikov) [inline server/bundles/io.cloudbeaver.server/src/io/cloudbeaver/server/events/WSUserSecretEventHandlerImpl.java]:
```suggestion
        if (!(activeUserSession instanceof WebSession)) {
           return;
        }
```

File: server/bundles/io.cloudbeaver.server/src/io/cloudbeaver/server/events/WSUserSecretEventHandlerImpl.java (hunk 1 above)
Comment (yagudin10) [reply server/bundles/io.cloudbeaver.server/src/io/cloudbeaver/server/events/WSUserSecretEventHandlerImpl.java]: removed

File: server/bundles/io.cloudbeaver.server/src/io/cloudbeaver/server/events/WSUserSecretEventHandlerImpl.java (hunk 2 above)
Comment (yagudin10) [reply server/bundles/io.cloudbeaver.server/src/io/cloudbeaver/server/events/WSUserSecretEventHandlerImpl.java]: fixed

**Pull request merged**